components/system/components/GlobalNotification.js

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/components/system/svg";
+import * as Strings from "~/common/strings";
+
+import { css } from "@emotion/react";
+
+import { DescriptionGroup } from "~/components/system/components/fragments/DescriptionGroup";
+
+const STYLES_NOTIFICATION_LIST = css`
+  position: fixed;
```

Comment:
I don't believe `display: flex;` and `flex-direction: column` are needed at all here. In fact it is redundant because block elements are naturally stacked on top of each other in a flow based layout system (How the DOM naturally behaves with `inline`, `inline-block`, and `block` elements. So basically all you need is position fixed.

There is also nothing wrong with having some default `right: 0; bottom: 0;` on this element since `inline-styles` always take priority over class styles.